security/medusa/l3/comm.c

```diff
@@ -4,12 +4,72 @@
 #include <linux/medusa/l3/arch.h>
 #include <linux/medusa/l3/registry.h>
 #include <linux/medusa/l3/server.h>
+#include <linux/medusa/l4/interface.h>
+
 #include "l3_internals.h"
 
+medusa_answer_t development_converter(authserver_answer_t code)
+{
+	medusa_answer_t retval;
+	switch(code) {
+		case AUTHS_ERR:
+			retval = MED_ERR;
+			break;
+		case AUTHS_ALLOW:
+		case AUTHS_DBG_ALLOW:
+		case AUTHS_FORCE_ALLOW:
+		case AUTHS_NOT_REACHED:
+			retval = MED_OK;
+			break;
+		case AUTHS_DENY:
+		case AUTHS_SKIP:
+			retval = MED_NO;
+			break;
+		default:
+			retval = MED_OK;
+	}
+	return retval;
+}
+
+medusa_answer_t production_converter(authserver_answer_t code)
+{
+	medusa_answer_t retval;
+	switch(code) {
+		case AUTHS_ERR:
+			retval = MED_ERR;
+			break;
+		case AUTHS_ALLOW:
+		case AUTHS_FORCE_ALLOW:
```

Comment:
should be MED_FORCE_ALLOW

Reply:
MED_YES is deprecated therefore cannot be used unless we will find workaround and start use it again